wpeverest/everest-forms
# Fix - Form builder design bug on wp.com dashboard

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes the builder design bug on wordpress.com dashboard.

### How to test the changes in this Pull Request:

1. Create a new form on wordpress.com dashboard.
2. Check the form builder design.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form builder design bug on wp.com dashboard
